data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -34,7 +34,8 @@ class ExtractTransformLoad:
     GEOID_TRACT_FIELD_NAME: str = "GEOID10_TRACT"
     # TODO: investigate. Census says there are only 217,740 CBGs in the US. This might be from CBGs at different time periods.
     EXPECTED_MAX_CENSUS_BLOCK_GROUPS: int = 250000
-    EXPECTED_MAX_CENSUS_TRACTS: int = 73076
+    # TODO: investigate. Census says there are only 73,057 tracts in the US. This might be from tracts at different time periods.
```

Comment:
Shouldn't this `TODO` be removed?

Reply:
No I just added it! EXPECTED_MAX_CENSUS_TRACTS is about 90 tracts higher than what census says we should expect.